File: frontend/src/app/model/announcement.ts
import { AnnouncementType } from "./announcementType";

export class Announcement {
    constructor(title: string, announcementType: AnnouncementType, announcementId: string) {
        this.title = title;
        this.announcementType = announcementType;
        this.announcementId = announcementId;
    }

    title: string;
    announcementType: AnnouncementType;
    announcementId: string;
}

File: src/main/java/com/bridge/faciliwise/model/Announcement.java
package com.bridge.faciliwise.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Announcement {
    String title;
    AnnouncementType announcementType;
    String announcementId;
}
